src/cmd-build

```diff
@@ -314,7 +341,16 @@ cat > tmp/meta.json <<EOF
 }
 EOF
 
-echo '{ "images": {} }' > tmp/images.json
+cat > tmp/images.json <<EOF
+{
+  "images": {
+    "ostree": {
+        "path": "ostree-commit.tar",
```

Comment:
We discussed this in IRC but I'll add the comment here too: can we add the name/version information into the filename here to match the other artifacts? i.e. `fedora-coreos-30.20190930.dev.2.ostree-commit.tar`